cvmfs/cvmfs_server

```diff
@@ -34,7 +34,7 @@ CVMFS_UPDATEGEO_HOUR=10 # First hour of day for update, 0-23, default 10am
 CVMFS_UPDATEGEO_MINDAYS=25 # Minimum days between update attempts
 CVMFS_UPDATEGEO_MAXDAYS=100 # Maximum days old before considering it an error
 
-CVMFS_UPDATEGEO_URLBASE="http://geolite.maxmind.com/download/geoip/database"
+CVMFS_UPDATEGEO_URLBASE="${CVMFS_UPDATEGEO_URLBASE:=http://geolite.maxmind.com/download/geoip/database}"
 CVMFS_UPDATEGEO_URLBASE6="${CVMFS_UPDATEGEO_URLBASE}/GeoLiteCityv6-beta"
```

Comment:
This could have been overridden in /etc/cvmfs/cvmfs_server_hooks.sh instead, which is the documented way.

Also, ":=" already assigns a default value to the variable being substituted as a side effect; it is more proper to use ":-".